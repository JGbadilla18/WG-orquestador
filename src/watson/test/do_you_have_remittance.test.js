const testWatson = require('./testWatson');

watsonText = 'Necesito enviar un remes'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});
