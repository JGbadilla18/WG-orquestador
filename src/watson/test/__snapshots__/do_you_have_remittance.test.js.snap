// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Necesito enviar un remes 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "buttonType": "web_url",
      "buttons": Array [
        Object {
          "title": "Aquí",
          "url": "https://enviosdedineroabancoindustrial.com/#enviar_dinero",
        },
      ],
      "response_index": 0,
      "response_type": "button",
      "text": "Conoce más de nuestras remesas.",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;
